xray/services/utils/ignorerulebody.go

```diff
@@ -42,12 +57,12 @@ type IgnoreFilterNameVersionPath struct {
 }
 
 type ExposuresFilterName struct {
-	Catagories []ExposuresCatagories `json:"catagories,omitempty"`
-	Scanners   []string              `json:"scanners,omitempty"`
-	FilePath   []string              `json:"file_path,omitempty"`
+	Categories []ExposureType `json:"categories,omitempty"`
+	Scanners   []string       `json:"scanners,omitempty"`
+	FilePath   []string       `json:"file_path,omitempty"`
 }
 
-type ExposuresCatagories struct {
+type ExposuresCategories struct {
```

Comment:
it seems that the only usage for this struct was in ExposuresFilterName. If this is indeed the case we can erase this struct

Reply:
its API used in different repo